build.sh

```diff
@@ -157,30 +209,90 @@ function run() {
     fi
 }
 
+function runAgentByClass() {
+    local agentExampleClass=$1
+    if [[ -z "${agentExampleClass}" ]]; then
+        err_log "error: try to run undefined agent class, empty class"
+        printHelp
+        exit 1
+    else
+        printf "example agent class: %s\n" "${agentExampleClass}"
+    fi
+
+    checkJava
+    if [[ "${OSTYPE}" =~ "linux"* ]] || [[ "${OSTYPE}" =~ "darwin"* ]]; then
+       printf "%s\n" "try to execute an agent example"
+    else
+        err_log "unknown OS type: \"${OSTYPE}\". Please coheck package ${JMC_DIR}/agent README.MD"
```

Comment:
Almost there, on re-reading I found this one: "coheck" should probably be "check"